From a69b567d97f7c9193924c775e1dd86e43a35b8bd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ludovic=20Court=C3=A8s?= <ludo@gnu.org>
Date: Sun, 7 Jul 2019 15:07:44 +0200
Subject: [PATCH 1/3] build: Do not record LDFLAGS in .pc file.

Fixes <https://bugs.gnu.org/36339>.
Reported by Cyprien Nicolas <cyprien@nicolas.tf>.

* configure.ac (GUILE_LIBS): Remove $LDFLAGS.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 9073deba0..92966ce34 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1688,7 +1688,7 @@ do
 done
 
 GUILE_CFLAGS="$GUILE_CFLAGS $PTHREAD_CFLAGS"
-GUILE_LIBS="$LDFLAGS $LIBS"
+GUILE_LIBS="$LIBS"
 
 AC_SUBST(GUILE_LIBS)
 AC_SUBST(GUILE_CFLAGS)
-- 
2.24.1

